api/envoy/extensions/http/header_validators/envoy_default/v3/header_validator.proto

```diff
@@ -0,0 +1,112 @@
+syntax = "proto3";
+
+package envoy.extensions.http.header_validators.envoy_default.v3;
+
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.http.header_validators.envoy_default.v3";
+option java_outer_classname = "HeaderValidatorProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/http/header_validators/envoy_default/v3;envoy_defaultv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Envoy's default Header Validator config]
+
+// This extension validates that HTTP request and response headers are well formed according to respective RFCs.
```

Comment:
RST ordered lists are a pita and prone to not being set correctly

i think this wants to be use formatting like:

```rst

This is an ordered list:

#. sadfsf
#. asdfsadf

```

not sure what is causing the list to collapse (https://storage.googleapis.com/envoy-pr/c7bb4e1/docs/api-v3/extensions/http/header_validators/envoy_default/v3/header_validator.proto.html#extensions-http-header-validators-envoy-default-v3-headervalidatorconfig)

perhaps no space between preceeding text and the list - i tested with https://livesphinx.herokuapp.com/ and i had to indent the list items - altho that doesnt seem necessary according to the sphinx docs https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks

either way, lets use hashes for the ordered list as this is more maintainable

Reply:
done